src/main/java/seedu/address/logic/commands/AddDeliveryCommand.java

```diff
@@ -32,7 +32,7 @@ public class AddDeliveryCommand extends Command {
             + PREFIX_DATETIME + "dd-mm-yyyy hh:mm "
             + PREFIX_SUPPLIER_INDEX + "SUPPLIER_INDEX "
             + PREFIX_PRODUCT + "PRODUCT "
-            + PREFIX_QUANTITY + "QUANTITY kg/g/litres/ml/units "
+            + PREFIX_QUANTITY + "QUANTITY kg/g/l/ml/units "
```

Comment:
I think litre should be represented by L and millilitre should be represented by mL